playwright-e2e/tests/dsm/miscellaneous/show-follow-up-survey.spec.ts

```diff
@@ -8,7 +8,7 @@ import { StudyName } from 'dsm/navigation';
 test.describe('Follow-Up Surveys', () => {
   let followupSurveyPage: FollowUpSurveyPage;
 
-  const studies = [StudyName.PANCAN, StudyName.ANGIO, StudyName.OSTEO, StudyName.LMS, StudyName.OSTEO2, StudyName.PROSTATE, StudyName.ESC];
+  const studies = [StudyName.ANGIO, StudyName.OSTEO, StudyName.LMS, StudyName.OSTEO2, StudyName.PROSTATE, StudyName.ESC];
```

Comment:
I believe you can also remove code referring to pancan family_history like below code
      case StudyName.PANCAN:
        survey = 'FAMILY_HISTORY  (NONREPEATING)';
        break;

Reply:
oh! good catch - taking a look